src/bin/scheduler/setup/rcc.rs

```diff
@@ -161,6 +139,74 @@ impl SetupStep for StepRCCCommand {
     }
 }
 
+struct StepDisableSharedHolotree {
+    config: GlobalConfig,
```

Comment:
As previously mentioned: I don't think adding `GlobalConfig` as a field is a good idea. (But let's discuss this for the other step only).

Reply:
Done